### DISCUSSION TITLE
Is there a way to create a full-width section?

### DISCUSSION THREAD

**Discussion by rajk-cerebras** [pradyunsg/furo#186] [Questions] [answered]
Hello - I use Sphinx Book Theme for a project and want to use your Furo theme for another project. Is there a way I can force the content to be full width by using in raw html something like `<div class="full-width">`? I tried it but it didn't work, but maybe I am not using the correct CSS element? Thanks for your answer!

**Comment by pradyunsg** [comment, marked as answer]
Could you clarify what you mean by "full-width section"?

**Reply by rajk-cerebras** [reply]
So, I did this in Sphinx Book Theme. I want to layout a 48-column table that extends fully into the right sidebar. I got this below:
![image](https://user-images.githubusercontent.com/74557030/125533890-a7a133d8-25b1-4fb8-b90a-11694e0859fc.png)
by inserting a `.. raw:: html` section as below:
![image](https://user-images.githubusercontent.com/74557030/125534112-94c67561-0db3-4f2c-ae51-a20b0341b7a5.png)

My question is, can I do this with Furo theme? Thanks!

**Reply by pradyunsg** [reply]
And… the answer to your question is no, you can’t do that.

**Reply by rajk-cerebras** [reply]
I see, thanks.
